style.css

```diff
@@ -0,0 +1,90 @@
+.dots {
+    width: 30px;
+    height: 40px;
+    position: relative;
+    left: 300px;
+    top: -170px;
+    background-image: radial-gradient(circle, black 4px, transparent 5px);
+    background-size: 100% 33.33%;
+}
+
+.texttile {
+    position: relative;
+    margin-left: 10px;
+    margin-right: 5px;
+    top: -155px;
+}
+
+.like {
+    position: relative;
+    left: 10px;
+    top: -153px
```

Comment:
Seeing small relative positioning tweaks (like `left: 10px`) is usually not concerning -- it's the big jumps and negative numbers here that led me to make the video to show you another way. I don't mean to suggest that _nothing_ ever be positioned relatively or absolutely -- just that it be used sparingly and not how you achieve a whole layout.